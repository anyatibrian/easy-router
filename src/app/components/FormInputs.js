import React from 'react'
import {TextInput } from 'react-native'

const FormInputs = (props) => {
    let userInput = null
    switch(props.type){
        case 'textinput':
            userInput = <TextInput {...props} />
            break
        default:
            return userInput
    }
    return userInput
}

export default FormInputs
