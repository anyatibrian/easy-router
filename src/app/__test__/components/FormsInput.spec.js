import React from 'react'
import { shallow } from 'enzyme'
import FormInputs from '../../components/FormInputs'

let wrapper
let instance
describe('MapsView component', () => {
    beforeEach(()=>{
        wrapper = shallow(<FormInputs/>)
        instance = wrapper.instance()
    })
    it("should test the component renders without fail", ()=>{
        expect(wrapper).toMatchSnapshot()
    })
})
