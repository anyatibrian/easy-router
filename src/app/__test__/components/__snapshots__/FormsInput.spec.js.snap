// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapsView component should test the component renders without fail 1`] = `ShallowWrapper {}`;
